### PR TITLE
ui completed

### PR DESCRIPTION
**Pull request by 333daz**
@nishintn 
@aswathym 

Hi,
Please find the pull request for the branch **dailyStatusUI** which has the UI for daily status page.

hosted url: http://333daz.github.io/DailyStatus/

### CODE REVIEW

File: javascript/status.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
+var i = 3;
```
Comment (nishintn) [inline javascript/status.js]: rename i.

File: javascript/status.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        var mmOptionElement = document.createElement("option");
+        var mm = (j < 10? "0": "") + j;
+        mmOptionElement.textContent = mm;
+        mmOptionElement.value = mm;
+        document.getElementById("minutePicker").appendChild(mmOptionElement);
+      }
+}
+
+
+function saveButtonPressed() {
```
Comment (nishintn) [inline javascript/status.js]: Rewrite logic as discussed, A statusHistory class which has all the UI elements. On click of save. create an obhect of the class/ set the properties and then add it to DOM.

**Comment by nishintn** [comment]
@333daz ,
Added some feedback.

**Comment by 333daz** [comment]
@nishintn ,
Added class for status history.

**Pull request merged**